bitnami/postgresql-ha/templates/postgresql/statefulset.yaml

```diff
@@ -533,6 +536,7 @@ spec:
             - name: custom-metrics
               mountPath: /conf
               readOnly: true
+          {{- end }}
```

Comment:
There is an issue with the if/end here, could you please apply this suggestion?
```suggestion
            {{- end }}
          {{- end }}
          {{- if .Values.metrics.customMetrics }}
```

This is to separate the conditional from `volumeMounts` and `command`/`args` when `metrics.customMetrics` are enabled.

This way, all if blocks should be properly closed:

```
          {{- if or .Values.postgresql.usePasswordFile .Values.metrics.customMetrics }} #1
          volumeMounts:
            {{- if .Values.postgresql.usePasswordFile }} #2
            - name: password
              mountPath: /opt/bitnami/postgresql/secrets/
            {{- end }} #2
            {{- if .Values.metrics.customMetrics }} #3
            - name: custom-metrics
              mountPath: /conf
              readOnly: true
            {{- end }} #3
          {{- end }} #1
          {{- if .Values.metrics.customMetrics }} #4
          {{- if .Values.diagnosticMode.enabled }} #5
          command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
          {{- end }} #5
          {{- if .Values.diagnosticMode.enabled }} #6
          args: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.args "context" $) | nindent 12 }}
          {{- else }} #6
          args: ["--extend.query-path", "/conf/custom-metrics.yaml"]
          {{- end }} #6
          {{- end }} #4
```